extensions/Worker.Extensions.ServiceBus/src/ServiceBusMessageActions.cs

```diff
@@ -1,4 +1,4 @@
-using System;
+﻿using System;
```

Comment:
This file is also missing a file header, let's take the opportunity to fix (will track making sure this is checked)